wordpress-mobile/WordPress-iOS
# Reader: Builds endpoints preserving desired version.

Fixes #5189 

To test:
Follow a feed (mashable, arstechnica) and a site. 
In your reader's following list, tap on the site name in a post card of both the feed and a site. 
Confirm that the feed and site loads correctly. 

Needs review: @koke
